extra-multimedia/feeluown-local/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME="feeluown-local"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for the local music source(s)"
+PKGDEP="feeluown python-3 marshmallow mutagen fuzzywuzzy"
+PKGBREAK="marshmallow<3.0.0 fuocore"
```

Comment:
You should actually break *and* replace an older fuocore version, defining the last version of fuocore available from the repository.

Also, use `<=` for marshmallow, that is, reference the last version before 3.0.0 that is available.

See the [Debian Policy Manual](https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-in-other-packages).